requirements/requirements.txt

```diff
@@ -0,0 +1,90 @@
+#
+# This file is autogenerated by pip-compile with Python 3.7
+# by the following command:
+#
+#    'make requirements'
+#
+accept-types==0.4.1
+    # via -r requirements/requirements.in
+aiohttp==3.8.4
+    # via -r requirements/requirements.in
+aiokafka==0.8.0
+    # via -r requirements/requirements.in
+aiosignal==1.3.1
+    # via aiohttp
+anyio==3.6.2
+    # via watchfiles
+async-timeout==4.0.2
+    # via
+    #   aiohttp
+    #   aiokafka
+asynctest==0.13.0
+    # via aiohttp
+attrs==22.2.0
+    # via
+    #   aiohttp
+    #   jsonschema
+avro @ https://github.com/aiven/avro/archive/f37a4ea69560d798cc718fccd7f75223527f5ebb.tar.gz#subdirectory=lang/py
+    # via -r requirements/requirements.in
+charset-normalizer==3.1.0
+    # via aiohttp
+frozenlist==1.3.3
+    # via
+    #   aiohttp
+    #   aiosignal
+idna==3.4
+    # via
+    #   anyio
+    #   yarl
+importlib-metadata==6.1.0
+    # via jsonschema
+isodate==0.6.1
+    # via -r requirements/requirements.in
+jsonschema==3.2.0
+    # via -r requirements/requirements.in
+kafka-python @ https://github.com/aiven/kafka-python/archive/1b95333c9628152066fb8b1092de9da0433401fd.tar.gz
+    # via
+    #   -r requirements/requirements.in
+    #   aiokafka
+multidict==6.0.4
+    # via
+    #   aiohttp
+    #   yarl
+networkx==2.6.3
+    # via -r requirements/requirements.in
+packaging==23.0
+    # via aiokafka
+protobuf==3.20.3
+    # via -r requirements/requirements.in
+pyrsistent==0.19.3
+    # via jsonschema
+python-dateutil==2.8.2
+    # via -r requirements/requirements.in
+six==1.16.0
+    # via
+    #   isodate
+    #   jsonschema
+    #   python-dateutil
+sniffio==1.3.0
+    # via anyio
+tenacity==8.2.2
+    # via -r requirements/requirements.in
+typing-extensions==4.5.0
+    # via
+    #   aiohttp
+    #   anyio
+    #   async-timeout
+    #   avro
+    #   importlib-metadata
+    #   yarl
+ujson==5.7.0
+    # via -r requirements/requirements.in
+watchfiles==0.19.0
+    # via -r requirements/requirements.in
+yarl==1.8.2
+    # via aiohttp
+zipp==3.15.0
+    # via importlib-metadata
+
+# The following packages are considered to be unsafe in a requirements file:
+# setuptools
```

Comment:
This warning happens because `jsonschema` specifies setuptools as a dependency. It looks like this is fixed in latest major version (4.*).